hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/SerialConsumer.java

```diff
@@ -52,7 +55,8 @@ public SerialConsumer(ReceiverFactory messageReceiverFactory,
                           MessageConverterResolver messageConverterResolver,
                           Topic topic,
                           ConfigFactory configFactory,
-                          OffsetQueue offsetQueue) {
+                          OffsetQueue offsetQueue,
+                          OAuthHandlers oAuthHandlers) {
```

Comment:
Renamed to `MessageDeliveryHandlersController` which might be or not related to security.